### ISSUE TITLE
Add headless run mode to oracle.

### ISSUE THREAD

**Issue by Daenyth** [Cockatrice/Cockatrice#1435]
Possible args:

```
--headless
--json-source # file:// or http[s]://
--token-source # file:// or http[s]://
--cards-output <path> # output path for cards.xml. If directory, add cards.xml. If matching *.xml, write to that path
--tokens-output <path> # likewise for tokens
--output <path> # directory to output both cards.xml and tokens.xml
--no-cards # Don't download or write cards
--no-tokens # Don't download or write tokens
-v | --verbose # Print CLI output for progress steps
```

I'm OK if we support this only for Qt5 compiled versions. Doing this much arg parsing in Qt4 by hand would be obnoxious.

**Comment by tooomm** [comment]
related comments: https://github.com/Cockatrice/Cockatrice/pull/1434#issuecomment-135905868

**Comment by Daenyth** [comment]
I'll leave that to be added as another ticket once this one is done.

**Comment by ZeldaZach** [comment]
We plan on replacing Oracle at some point, so this would be wasted efforts
